dan_layer/storage/src/consensus_models/block.rs

```diff
@@ -766,20 +773,20 @@ impl Block {
                     // because the engine can never emit such a substate diff.
                     if substate.created_by_transaction == transaction.id {
                         updates.push(SubstateUpdate::Create(SubstateCreatedProof {
-                            created_qc: substate.get_created_quorum_certificate(tx)?,
+                            // created_qc: substate.get_created_quorum_certificate(tx)?,
                             substate: substate.into(),
                         }));
                     } else {
                         updates.push(SubstateUpdate::Destroy(SubstateDestroyedProof {
                             substate_id: substate.substate_id.clone(),
                             version: substate.version,
-                            justify: QuorumCertificate::get(tx, &destroyed.justify)?,
+                            // justify: QuorumCertificate::get(tx, &destroyed.justify)?,
```

Comment:
is this ok to do?

Reply:
This is only used when calling RPC block sync, which has been replaced by state sync. The indexer still uses block sync however, but doesn't validate the justify anyway. This part needs to be rethought so easier to leave this out.